test/EFCore.PG.FunctionalTests/Query/NetworkQueryNpgsqlTest.cs

```diff
@@ -80,7 +80,7 @@ public void IPAddress_inet_parse_column()
                            .Where(x => x.Inet.Equals(IPAddress.Parse(x.TextInet)))
                            .ToArray();
 
-                AssertContainsSql("WHERE x.\"Inet\" = CAST(x.\"TextInet\" AS inet)");
+                AssertContainsSql("WHERE (x.\"Inet\" = CAST(x.\"TextInet\" AS inet))");
```

Comment:
What changed to add these now?

Reply:
The full WHERE clause in 3.0 is `WHERE (x."Inet" = CAST(x."TextInet" AS inet)) OR (x."Inet" IS NULL AND x."TextInet" IS NULL)`. IIRC the null checking clause (after the OR) is new, so the first clause is enclosed in parentheses...

Reply:
Oh... Did any of the configurations change to enable/disable use relational nulls?

Reply:
Also, if that's the full `WHERE` now, could you update these tests to include it in the assert? (That way if anything changes again...)

Reply:
As a general rule, the SQL assertions currently only assert the minimal needed to know that on our side the translation occurred correctly, rather than include more SQL that isn't related to our translation (like the 2nd null-checking clause). This comes from my experience in tracking upstream EF Core, where the SQL generated changes quite frequently in small ways, leading to many uninteresting test failures on our side...

Reply:
Hmm... That's a fair point, considering that these tests only need to verify expression translation, not the relational null handling bit. Consider this resolved, thanks!